frontend/toolkit-webapp/src/pages/ForgotPassword.js

```diff
@@ -1,8 +1,81 @@
-function ForgotPassword() {
+import React, { useState } from 'react'
+import { Button, Container, TextField, makeStyles, Typography, Paper, Box } from '@material-ui/core'
+import Avatar from '@material-ui/core/Avatar'
+import PermIdentityOutlinedIcon from '@material-ui/icons/PermIdentityOutlined';
+import config from '../config.json'
+import resetPassStyle from '../styles/registerStyle'
+// import '../css/Registration.css';
+import TopNavBar from '../components/TopNavBar'
+import videoSource from '../img/PEOSTRI.mp4'
+
+const ForgotPassword = (props) => {
+    
+    const [email, setEmail] = useState('')
+    const classes = resetPassStyle()
+
+    const onLogin = (e) => {
+        props.history.push('login');
+    }
+
+    const resetLink = async (e) => {
```

Comment:
Clicking on button with blank text field should not be a possibility. Prompt user to enter a valid email format.

Reply:
Fixed it! Thanks